### ISSUE TITLE
Provide a gen_climate function using an import function (as in Javatics)

### ISSUE THREAD

**Issue by VEZY** [SticsRPacks/SticsRFiles#118]
See `gen_climate.R` function (maybe export it? 
https://github.com/SticsRPacks/SticsRFiles/blob/main/R/gen_climate.R

**Comment by plecharpent** [comment]
Yes it would be of interest, but for the moment this function is doing only a file concatenation, which are supposed to contain
Stics formatted data

It would be interesting to take into account import and conversions functionalities as in JavaStics to create Stics
weather files from any file containing weather data
